### PR TITLE
Update READMEs for CMake and MSVC

### PR DESCRIPTION
**Pull request by DinCahill**
Please review and suggest improvements.

Indeed, with CMake, you don't need gmake on FreeBSD! Amazing.

### CODE REVIEW

File: README.VisualStudio.md [hunk 1]
```diff
@@ -2,16 +2,44 @@
...
+Now you can sit back and relax whilst this script downloads the binary versions of all the libraries.
+
+Usage: `.\WindowsBuilder.ps1 [-deps] [-playd] -arch x86|x64 [<CommonParameters>]`
+
+Full help text: `Get-Help .\Windows-Builder.ps1`
+
+The build will be in a directory like: `x86\build\Release\` along with all the necessary DLLs.
+
+**Note**: Only the **Release** configuration is currently supported.
+Until that's supported, you're on your own. Use Linux or something.
```
Comment (ddm999) [inline README.VisualStudio.md]: "Until that's supported": you haven't actually mentioned what 'that' refers to - a better line would be something like "If you want to use other configurations, you'll have to use Linux or something."

File: README.md [hunk 2]
```diff
@@ -57,33 +57,96 @@ do weird things in the presence of Telnet-isms.
...
 
-* [libuv] 1.9.1+
-* [SDL2] 2.0.3
+* [libuv] 1.9.1+;
+* [SDL2] 2.0.3;
 * A C++14 compiler (recent versions of [clang], [gcc], and Visual Studio
-  work)
-* [CMake] 2.8+, for building.
+  work);
+* [CMake] 2.8+, for building [optional, but recommended].
```
Comment (ddm999) [inline README.md]: "optional, but recommended" should be in (round brackets), not [square ones]

File: README.VisualStudio.md (hunk 1 above)
Comment (CaptainHayashi) [inline README.VisualStudio.md]: Presumably he meant `Until Debug's supported`, but yours reads better :p

File: README.md (hunk 2 above)
Comment (CaptainHayashi) [inline README.md]: +1

File: README.md [hunk 3]
```diff
@@ -57,33 +57,101 @@ do weird things in the presence of Telnet-isms.
...
+
+#### FreeBSD (10+)
+
+	root@freebsd:/ # pkg install libmpg123 libsndfile libuv sdl2
+    
+Additional dependencies for building without CMake:
+
+	root@freebsd:/ # pkg install gmake pkgconf
+
+#### OS X
```
Comment (CaptainHayashi) [inline README.md]: should be `macOS` now, son

File: README.VisualStudio.md [hunk 4]
```diff
@@ -2,16 +2,59 @@
 
 Here are some field notes for building `playd` with Microsoft Visual Studio.
 
-Visual Studio builds used to be much harder to pull off, but nowadays most
-of the dependencies are nicely packaged.  _Most_.
+Visual Studio builds used to be much harder to pull off, but nowadays all
+of the dependencies are nicely packaged for both x86 and x64.
+
+**Note**: The x64 build currently has a few warnings, but it appears to work!
```
Comment (DinCahill) [inline README.VisualStudio.md]: Yes. I'll let you know on IRC.

**Pull request merged**